Sources/Sentry/SentrySdkInfo.m

```diff
@@ -1,25 +1,30 @@
 #import "SentrySdkInfo.h"
 #import <Foundation/Foundation.h>
 
+/**
+ * This is required to identify the package manager used when installing sentry.
+ */
+#if SWIFT_PACKAGE
+static NSString *SENTRY_PACKAGE_INFO = @"spm:getsentry/%@";
```

Comment:
`m`: I would prefer adding tests for these constants here. I think you need to restructure the code a bit to make it work. As we use them in `NSString stringWithFormat` a small typo could easily lead to a crash we only detect after releasing.